streams/src/test/java/org/apache/kafka/streams/tools/StreamsResetterTest.java

```diff
@@ -247,6 +278,32 @@ public void shouldDetermineInternalTopicBasedOnTopicName1() {
         assertTrue(streamsResetter.matchesInternalTopicFormat("appId-KTABLE-FK-JOIN-SUBSCRIPTION-REGISTRATION-12323232-topic"));
     }
 
+    @Test
+    public void testResetToDatetimeWhenPartitionIsEmptyResetsToLatestOffset() {
+        final long[] beginningAndEndOffsets = {0L, 5L, 10L, 20L};
```

Comment:
So ensure that we really set to end, beginning and end should be set to different values, otherwise, we cannot distinguish both. Thus, similar to the test above, testing a single combination beginning=5 and end=10 seems sufficient?

Reply:
On this one, seems to me that only one value is sufficient. As the test describes `testResetToDatetimeWhenPartitionIsEmptyResetsToLatestOffset`, the partition is empty - thus beginning offset == end offset. I will change to have a beginning offset equals to 5 anyways.

Reply:
Fair enough :)